### DISCUSSION TITLE
Filter collections by parent?

### DISCUSSION THREAD

**Discussion by ivasilov** [vendurehq/vendure#1140] [General]
Hello,

I would like to build dropdown for nested categories in the storefront. I would like to add `parent` field to [CollectionFilterParameter](https://www.vendure.io/docs/graphql-api/shop/input-types/#collectionfilterparameter) which is used by [collections](https://www.vendure.io/docs/graphql-api/shop/queries/#collections) API call. If the parameter is undefined (but is present in the query variables), only the top level categories should be returned.

Would it be ok if I add the `parent` parameter in a PR?

**Comment by michaelbromley** [comment]
Hi,

Have you considered using the `collection.children` field? This already exists and will give an array of all children of the selected Collection.
